OpenEdXMobile/src/main/java/org/edx/mobile/view/MainTabsDashboardFragment.java

```diff
@@ -128,7 +129,9 @@ public void onFragmentSelected() {
                     }
                 }));
 
-        if (environment.getConfig().getProgramConfig().isEnabled()) {
+        if (environment.getConfig().getProgramConfig().isEnabled() &&
+                (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT ||
```

Comment:
It should have been more better, if you add the KitKat OS flag within
`getProgramConfig().isEnabled()`